models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -0,0 +1,84 @@
+# Description
+A neural architecture search algorithm for volumetric (3D) segmentation of the pancreas and pancreatic tumor from CT image.
+
+# Model Overview
+This model is trained using the state-of-the-art algorithm [1] of the "Medical Segmentation Decathlon Challenge 2018" with 196 training images, 56 validation images, and 28 testing images.
+
+## Data
+The training dataset is Task07_Pancreas.tar from http://medicaldecathlon.com/. And the data list/split can be created with the script `scripts/prepare_datalist.py`.
+
```

Comment:
Hi @dongyang0122 , could you please also add an example command here? For, example, to match the dataset json path mentioned in `train.yaml`, it can be:

```
python scripts/prepare_datalist.py --path /path-to-Task07_Pancreas/ --output configs/dataset_0.json
```